src/test/java/nl/tudelft/contextproject/saveLoad/ApplicationSettingsTest.java

```diff
@@ -0,0 +1,125 @@
+    
+    /**
+     * Asserts that the specified settings are the default settings.
+     * @param settings Settings to be tested.
+     */
+    private static void assertDefaultSettings(ApplicationSettings settings) {
+        assertEquals(ApplicationSettings.DEFAULT_RESX, settings.getRenderResX());
+        assertEquals(ApplicationSettings.DEFAULT_RESY, settings.getRenderResY());
+        assertEquals(ApplicationSettings.DEFAULT_VLC_LOC, settings.getVlcLocation());
+        assertEquals(new HashMap<Integer, String>(), settings.getAllCameraIPs());
+    }
+
+    /**
+     * Tests {@link ApplicationSettings#isLoaded()}.
+     */
+    @Test
+    public void testIsLoaded() throws Exception {
+        ApplicationSettings settings = spy(ApplicationSettings.getInstance());
+        File invalidFile = new File("src/test/resources/settingsLoadInvalidTest.txt");
+        File validFile = new File("src/test/resources/settingsLoadTest.txt");
+        whenNew(File.class).withAnyArguments().thenReturn(invalidFile, validFile);
+        assertFalse(settings.load());
+        assertFalse(settings.isLoaded());
+        //First file has invalid settings, second file has valid settings.
+        assertTrue(settings.load());
+        assertTrue(settings.isLoaded());
+    }
+    
+    /**
+     * Tests {@link ApplicationSettings#reset()}.
+     */
+    @Test
+    public void testReset() {
+        ApplicationSettings settings = ApplicationSettings.getInstance();
+        settings.reset();
+        assertDefaultSettings(settings);
+    }
+
+    /**
+     * Tests {@link ApplicationSettings#load()}.
+     * Tries to load a non-existent file.
+     */
+    @Test
+    public void testLoadNoFile() throws Exception {
+        ApplicationSettings settings = spy(ApplicationSettings.getInstance());
+        settings.setRenderResolution(0, 0);
+        doThrow(new FileNotFoundException()).when(settings, "getScanner");
+        assertFalse(settings.load());
+        assertFalse(settings.isLoaded());
+    }
+    
+    @Test
+    @SuppressWarnings("PMD.AvoidUsingHardCodedIP")
+    public void testLoadFull() throws Exception {
+        ApplicationSettings settings = spy(ApplicationSettings.getInstance());
+        File file = new File("src/test/resources/settingsLoadTest.txt");
+        whenNew(File.class).withAnyArguments().thenReturn(file);
+        settings.load();
+        HashMap<Integer, String> expectedIPs = new HashMap<Integer, String>();
+        expectedIPs.put(1, "65.65.65.65");
+        expectedIPs.put(3, "420.420.420.420");
+        assertEquals(1965, settings.getRenderResX());
+        assertEquals(420, settings.getRenderResY());
+        assertEquals("D:\\program files", settings.getVlcLocation());
+        assertEquals(expectedIPs, settings.getAllCameraIPs());
+        
+    }
+
+    /**
+     * Tests {@link ApplicationSettings#save()}.
+     */
+    @Test
+    @SuppressWarnings("PMD.AvoidUsingHardCodedIP")
+    public void testSave() throws Exception {
```

Comment:
You are suppressing warnings in some of your tests. You may want to clarify why SuppressWarnings is used.